src/modules/45-dashboards/components/AidaChatInput/__tests__/AidaChatInput.test.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { act, fireEvent, render, RenderResult, screen } from '@testing-library/react'
+import { TestWrapper } from '@common/utils/testUtils'
+import AidaChatInput from '..'
+
+const renderComponent = (onEnter: (value: string) => void = jest.fn()): RenderResult =>
+  render(
+    <TestWrapper>
+      <AidaChatInput onEnter={onEnter} />
+    </TestWrapper>
+  )
+
+describe('AidaChatInput', () => {
+  test('it should display placeholder text in input', async () => {
+    const { container } = renderComponent()
+    expect(screen.getByPlaceholderText('common.csBot.askAIDA')).toBeInTheDocument()
+    expect(container.querySelector('svg')?.getAttribute('data-icon')).toEqual('arrow-right')
+  })
+
+  test('it should allow text entry within input', async () => {
+    renderComponent()
+
+    const input = screen.getByPlaceholderText('common.csBot.askAIDA') as HTMLInputElement
+
+    const mockText = 'dummy text'
+    await act(async () => {
+      fireEvent.change(input, {
+        target: { value: mockText }
+      })
+    })
+
+    expect(input.value).toBe(mockText)
```

Comment:
`toHaveValue` is great for places like this.
```suggestion
    expect(input).toHaveValue(mockText)
```